conf/httpd.conf.d/httpd.portal

```diff
@@ -201,10 +201,32 @@ $MaxClients = pf::services::manager::httpd::calculate_max_clients(pf::services::
 $StartServers = "4";
 $MinSpareServers = "2";
 
-if( pf::config::isenabled ($PfConfig->{services}{httpd_mod_qos})) {
+if( pf::config::isenabled ($PfConfig->{captive_portal}{httpd_mod_qos})) {
     my $qos = $MaxClients * .7;
     $QS_SrvMaxConnClose = $qos;
-    $QS_SrvMaxConnPerIP = $PfConfig->{services}{httpd_mod_qos_maximum_connections_per_device};
+    $QS_SrvMaxConnPerIP = $PfConfig->{captive_portal}{httpd_mod_qos_maximum_connections_per_device};
+}
+
+if( pf::config::isenabled ($PfConfig->{captive_portal}{httpd_mod_evasive})) {
+    if( pf::config::os_detection() eq 'debian' ) {
```

Comment:
It looks like no, we have that in the source code of mod_evasive:
#define DEFAULT_HASH_TBL_SIZE   3079ul  // Default hash table size
#define DEFAULT_PAGE_COUNT      2       // Default max page hit count/interval
#define DEFAULT_SITE_COUNT      50      // Default max site hit count/interval
#define DEFAULT_PAGE_INTERVAL   1       // Default 1 second page interval
#define DEFAULT_SITE_INTERVAL   1       // Default 1 second site interval
#define DEFAULT_BLOCKING_PERIOD 10      // Default block time (Seconds)
#define DEFAULT_LOG_DIR         "/tmp